packages/common/src/components/UvText/UvText.tsx

```diff
@@ -205,6 +207,32 @@ const UvText: React.FC<Props> = ({ question }) => {
     return 'Ej angivet'
   }
 
+  const getMedicalInvestigationValue = (valueList: ValueMedicalInvestigationList) => {
+    return (
+      <table className="ic-table iu-fullwidth">
+        <thead>
+          <tr>
+            <th scope="col">Ange utredning eller underlag</th>
+            <th scope="col">Datum</th>
+            <th scope="col">Från vilken vårdgivare</th>
+          </tr>
+        </thead>
+        <tbody>
+          {(valueList.list as ValueMedicalInvestigation[]).map((medicalIvestigation) => {
+            if (medicalIvestigation.informationSource.text === undefined) return null
+            return (
+              <tr>
+                <td>{medicalIvestigation.investigationType.code}</td>
```

Comment:
We should resolve this from `typeText` in the config so that we present the readable text instead.

![image](https://user-images.githubusercontent.com/28404/207287873-bc40b967-d32e-499b-8d2c-e3190c32a287.png)

Reply:
ofc!